packages/express/src/index.ts

```diff
@@ -15,92 +15,78 @@ export type Middleware = (
   next: NextFunction
 ) => Promise<void>;
 
-export default class LogtoClient {
-  constructor(private readonly config: LogtoExpressConfig) {}
-
-  handleSignIn =
-    (redirectUri = `${this.config.baseUrl}/logto/sign-in-callback`): Middleware =>
-    async (request, response) => {
-      const nodeClient = this.createNodeClient(request, response);
-      await nodeClient.signIn(redirectUri);
-    };
-
-  handleSignInCallback =
-    (redirectTo = this.config.baseUrl): Middleware =>
-    async (request, response) => {
-      const nodeClient = this.createNodeClient(request, response);
+const createNodeClient = (
+  request: IncomingMessage,
+  response: Response,
+  config: LogtoExpressConfig
+) => {
+  // We assume that `session` is configured in the express app, but need to check it there.
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (!request.session) {
+    throw new LogtoExpressError('session_not_configured');
```

Comment:
nit: maybe `session not found`?

Reply:
Also why do we need to maintain a new error class `LogtoExpressError`? Can we re-use `LogtoClientError`?

Reply:
1. `session not found` sounds like the user has no session, `not configured` is more accurate.
2. `LogtoClientError` don't have this error key.

Reply:
Well, OK